sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -81,3 +85,73 @@
     'ResourceTypes',
     'AccountPermissions',
 ]
+
+
+def upload_blob_to_url(
+        blob_url,  # type: str
+        data,  # type: Union[Iterable[AnyStr], IO[AnyStr]]
+        overwrite=False,  # type: bool
+        max_connections=1,  # type: int
+        encoding='UTF-8', # type: str
+        credential=None,  # type: Any
+        **kwargs):
+    """Upload data to a given URL
+
+    The data will be uploaded as a block blob.
+
+    :param str blob_url:
+        The full URI to the blob. This can also include a SAS token.
+    :param data:
+        The data to upload. This can be bytes, text, an iterable or a file-like object.
+    :type data: bytes or str or Iterable
+    :param bool overwrite:
+        Whether the blob to be uploaded should overwrite the current data.
+        If True, upload_blob_to_url will silently overwrite the existing data. If set to False, the
```

Comment:
I would remove the "silently". And should it be "overwrite the current blob"? Or is this intended to indicate that properties of an existing blob will be retained? If so, I would consider explicitly stating that...